crm_claim/README.rst

```diff
@@ -53,6 +53,7 @@ Contributors
 
 * Odoo S.A.
 * Vicent Cubells <vicent.cubells@tecnativa.com>
+* Bhavesh Odedra <bodedra@ursainfosystems.com>
```

Comment:
Don't put yourself as contributor because your contribution is purely stylistic and doesn't add/fix useful content.